### DISCUSSION TITLE
OTA Flash to ESPHome powered Smart-Plug

### DISCUSSION THREAD

**Discussion by Jannis87** [ESPresense/ESPresense#1045] [Ideas]
Hi there, 

I am wondering if it is possible to flash ESPresence on a device that is already running ESPHome or Tasmota. I just stumbled across the following product:
https://de.athom.tech/blank-1/esphome-eu-plug 

It is pre-flashed with Tasmota or ESPHome. Would be great if you could flash it over the air (via ESPHome/Tasmota Web UI?) without any soldering.

**Comment by ShonP40** [comment]
I was able to flash a different firmware on a Tasmota device by first flashing `tasmota-minimal` on it.
Could work with ESPresense as well.

**Reply by PoebelPogge** [reply]
Have you done any soldering or was a USB plug still available?

**Reply by ShonP40** [reply]
Flashed it through the Tasmota webui

**Comment by DTTerastar** [comment]
That device is a ESP8285.  It doesn't have any bluetooth so it wouldn't work
